zkevm-circuits/src/pi_circuit.rs

```diff
@@ -1509,6 +1509,11 @@ impl<F: Field> SubCircuit<F> for PiCircuit<F> {
                                     .collect_vec(),
                             ),
                             (TxFieldTag::IsCreate, tx.is_create.to_le_bytes().to_vec()),
+                            (TxFieldTag::TxInvalid, tx.invalid_tx.to_le_bytes().to_vec()),
+                            (
+                                TxFieldTag::AccessListGasCost,
+                                tx.access_list_gas_cost.to_le_bytes().to_vec(),
```

Comment:
Looks like `access_list_gas_cost` is under-constraints and prover can fill any value, because this pseudo field is not in normal specs so its validity is not assure.

Reply:
@hero78119  Could you provide more details for "access_list_gas_cost is under-constraints and prover can fill any value" and suggest a change?

Reply:
Because the logic to define `access_list_gas_cost` is not constrainted in plonk circuit. 

Here attach an example of [init_code_gas_cost](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/execution/begin_tx.rs#L153-L158) which is constrainted in circuit as constant. It seems to me that `access_list_gas_cost` can remove from transaction field/table and just derived it in begin_tx circuit logic?